### PR TITLE
Update intro-to-graphql.md

### PR DESCRIPTION
**Pull request by zamkevich**
Added explanation about search operators

### CODE REVIEW

**Comment by zamkevich** [comment]
Looks like i don't need a changeset for docs?

File: docs/guides/intro-to-graphql.md [hunk 1]
```diff
@@ -308,7 +308,7 @@ query {
 
 ### `search`
 
-Will search the list to limit results.
+Will search the list to limit results. It can search only by [labelField](/docs/api/create-list.md#labelfield).
```
Comment (timleslie) [inline docs/guides/intro-to-graphql.md:311]: This isn't quite true. At the moment the `search` is applied to a `Text` field called `name`. This is not currently configurable, although that is a feature which is on our backlog.

**Review by timleslie — CHANGES_REQUESTED** [review]

**Review by zamkevich — COMMENTED** [review]
@timleslie I changed it to:

"For now, it can search only by `name` field. The 'name' field must be a 'Text' type."

**Review by timleslie — APPROVED** [review]
👍 Thanks @zamkevich, this will hopefully clear things up a bit for readers of this documentation.

**Pull request merged**